music_album.rb

```diff
@@ -0,0 +1,20 @@
+class MusicAlbum
+  attr_accessor :can_be_archived, :on_spotify, :album_name, :genre
+  attr_reader :id
+
+  def initialize(_id, album_name, on_spotify, can_be_archived, genre)
+    @id = generate_random_id
+    @can_be_archived = can_be_archived
+    @on_spotify = on_spotify
+    @album_name = album_name
+    @genre = genre
+  end
+
+  def can_be_archived?
+    @can_be_archived
+  end
```

Comment:
:red_circle: [REQUIRED] Kindly make sure you correct this method as suggested by the previous reviewer. 

> This method should:
> 
>     * return true if parent's method returns true AND if on_spotify equals true
> 
>     * otherwise, it should return false
> 
> 
> Something like this should do the job:
> 
> ```
>     def can_be_archived?
>       super && @on_spotify == true
>     end
> ```
> 
> **Note:** The @can_be_archived property is not present in the class diagram, I suggest you remove it from all classes where it was added.